jobs/JGLOBAL_WAVE_POST_PNT

```diff
@@ -24,11 +21,10 @@ export DOBNDPNT_WAV='NO'  #not boundary points
 
 
 # Execute the Script
-${SCRgfs}/exgfs_wave_post_pnt.sh
-err=$?
-if [ ${err} -ne 0 ]; then
-  echo "FATAL ERROR: ex-script of GWES_POST failed!"
-  exir ${err}
+${SCRgfs}/exgfs_wave_post_pnt.sh && true
+export err=$?
+if [[ ${err} -ne 0 ]]; then
+  err_chk "FATAL ERROR: ex-script of GWES_POST failed!"
```

Comment:
same comment as before. And, `GWES_POST` is misleading.

Reply:
Maybe JGLOBAL_WAVE_POST_PNT failed?

Reply:
Thanks @DavidHuber-NOAA  - I know you were just moving what was already there.  Thanks for making this cleaner!